design-library/src/main/java/com/what3words/design/library/ui/components/VoiceAnimation.kt

```diff
@@ -70,10 +71,12 @@ object VoiceAnimationDefaults {
     @Composable
     fun defaultColors(
         orbColors: Color = MaterialTheme.w3wColorScheme.brand,
+        idleOrbColors: Color = MaterialTheme.colorScheme.errorContainer,
```

Comment:
@QuocNguyen-w3w in here should be MaterialTheme.w3wColorScheme.brand and you then should override in the AAOS app with errorContainer color, because if you see the component in the design library it uses brand for both 👍